starter/src/app/home/home.component.ts

```diff
@@ -21,7 +21,7 @@ import { Title } from "./title";
 	/**
 	 * Every Angular template is first compiled by the browser before Angular runs it's compiler.
 	 */
-	templateUrl: "./home.component.html"
+	template: require("./home.component.html")
```

Comment:
There is no need to use require anymore. This issue is solved in this PR #252 
We should use templateUrl instead ;)